clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -55,7 +55,16 @@ def __init__(self, cluster: str, pool: Optional[str]) -> None:
 
     def reload_state(self) -> None:
         logger.info('Reloading nodes')
-        kubernetes.config.load_kube_config(staticconf.read_string(f'{self.kubeconfig_path}'))
+        try:
+            kubernetes.config.load_kube_config(staticconf.read_string(f'{self.kubeconfig_path}'))
+        except TypeError:
+            error_msg = (
+                'Could not load KUBECONFIG; is this running on Kubernetes master?\n'
+                'Hint: if this is from the CLI, try using the clusterman-k8s-<clustername> wrapper script!'
```

Comment:
nit: Same comment here on Yelp-isms. For Yelp users, I'd suggest running an `os` command to check whether or not the wrapper exists before append this hint.

Reply:
fixed.